lib/chrome.js

```diff
@@ -21,6 +21,15 @@ export class StorageChangeEvent {
     this.listeners.push(callback);
   }
 
+  removeListener(callback) {
+    for (let i=0; i<this.listeners.length; i++) {
+      if (this.listeners[i] === callback) {
+        this.listeners.splice(i, 1);
+        break;
+      }
+    }
```

Comment:
I'd naturally use Array.filter for this but this approach is probably faster as it breaks early.